src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1452,12 +1452,6 @@ public static decimal Truncate(decimal d)
             return decimal.Truncate(d);
         }
 
-        public static unsafe double Truncate(double d)
```

Comment:
We can keep these methods and decorate them with `[Intrinsic]`. This will avoid adding a new FCall. We have recently removed a similar FCall for Abs: https://github.com/dotnet/runtime/commit/0b307c95286a4f468fbb627a66515054ff18dc87. This fallback code path is only used by reflection and interpreter -like scenarios.